test cases/common/121 shared module/meson.build

```diff
@@ -1,5 +1,14 @@
 project('shared module', 'c')
 
+# Windows on ARM doesn't appear to support the toolhelp API we use to emulate
+# runtime symbol resolution in this test.
+#
+# XXX: This should be written as a check if the macro
+# WINAPI_FAMILY_PARTITION(WINAPI_PARTITION_DESKTOP) evaluates to true?
+if host_machine.system() == 'windows' and ['aarch64', 'arm'].contains(host_machine.cpu_family())
+  error('MESON_SKIP_TEST windows on ARM does not support this test.')
+endif
```

Comment:
Loading of modules is done via `LoadPackagedLibrary`, and then to run it, you need to bundle the DLLs as assets (that part is not needed because we can't run cross-compiled tests anyway).

Reply:
The issue here isn't `LoadLibrary()` not being available for UWP, but the toolhelp API, which we use to grovel over all loaded modules, looking at their exported symbols, to emulate run-time linkage.  I rewrote this a bit so it does a more specific test for that.

Reply:
Makes sense :)